README.md

```diff
@@ -356,89 +423,431 @@ tondev tonos-cli update
+    --help, -h        Show command usage
+    --dictionary, -d  Mnemonic dictionary
+                      0  TON
+                      1  English
+                      2  Chinese Simplified
+                      3  Chinese Traditional
+                      4  French
+                      5  Italian
+                      6  Japanese
+                      7  Korean
+                      8  Spanish
+    --force, -f       Overwrite signer if already exists
+```
+**Note:** By default the dictionary is set to english, which allows using seed phrases generated by other TONOS tools, such as tonos-cli.
 
-#### Add Command
 
-Add a signer with an existing secret key or a mnemonic phrase.
+#### List registered signers
 
-#### Delete Command
+This command lists all registered signers with their public keys.
 
-Deletes signer from store.
+```bash
+tondev signers list
+```
 
-#### List Command
+Result:
 
-Show all signer in the registry. Only non secret data are shown.
+```bash
+$ tondev signers list
 
-#### Get Command
+Signer           Public Key
+---------------  ----------------------------------------------------------------
+sign1 (Default)  cffd3a2f1d241807b2205220a7d6df980e67a3cc7c47eba2766cdc1bbddfc0e3
+sign2            0fc4e781720d80f76257db333c6b6934090562418652cf30352878c87707aa94
+```
 
-Prints full information for the specified signer including all secret data.
+#### Get signer details
 
-#### Default Command
+This command lists all information (including secret data) for a specified signer.
 
-Sets specified signer as the default.
+```bash
+tondev signers get signer_name
+```
+
+Result:
+
+```bash
+$ tondev signers get sign2
+{
+    "name": "sign2",
+    "description": "",
+    "keys": {
+        "public": "760d69964d038997d891fca0a0407c2ffefb701e7cb2f9ff0a87fbbf1e8098f2",
+        "secret": "72571b5a9392e6bb215b460ca3c0545c34d790e185f66f5b2e7564329ffea86c"
+    }
+}
+```
+
+#### Set default signer
+
+This command sets a previously added signer as default (initially the first added signer is used by default).
+
+```bash
+tondev signers default signer_name
+```
+
+#### Delete a signer
+
+This command deletes a previously added signer from signer registry.
+
+```bash
+tondev signers delete signer_name
+```
 
 ### Network Registry
 
 Networks registry is a convenient way to organize all of your network configurations in one place.
 
 You can register several blockchains (networks) under short names 
-and then use this names as a target blockchain to play with contracts.  
+and then use these names as a target blockchain when working with contracts.  
 
-You can mark one of the network as a default.
+You can mark one of the networks as a default.
 It can be used in network commands without providing net name. 
 
-#### Add Command
+#### Add a network
 
-Add network with specified endpoints to the registry.
+This command adds a network to the tondev registry.
 
-#### Delete Command
+```bash
+tondev networks add network_name network_endpoint
```

Comment:
Add one more sample where you add several endpoints for the same network - for more reliability, for instance
add `mainnet` network with main2.ton.dev, main3.ton.dev, main4.ton.dev endpoints.